bundles/plugins/org.bonitasoft.studio.connectors/src/org/bonitasoft/studio/connectors/ui/property/section/ConnectorSection.java

```diff
@@ -419,5 +429,16 @@ public void refresh() {
         super.refresh();
         refreshTree();
     }
+    
+    protected ConnectorWizard createConnectorWizard(){
+		return new ConnectorWizard(getEObject(), getConnectorFeature(), getConnectorFeatureToCheckUniqueID());
+	}
+    
+    /**
+	 * @param connectorWizard
+	 */
+	protected void setConnectorEvent(ConnectorWizard connectorWizard,String connectorEvent) {
+		connectorWizard.setConnectorEvent(connectorEvent);
```

Comment:
The design is awkward. Maybe adding a connecorEvent in the wizard constructor can be simpler.